src/sql/workbench/contrib/charts/common/interfaces.ts

```diff
@@ -36,6 +39,16 @@ export enum ChartType {
 	Scatter = 'scatter'
 }
 
+export const ChartTypeToChartJsType: { [key in ChartType]: chartjs.ChartType } = {
```

Comment:
Is there a reason you're not using a function here and below? That's generally nicer since you can handle situations like an unknown chart type being passed in better.

Reply:
Yeah, because that forces us to keep both ChartType and ChartTypeToChartJsType consistent with each other. Also, it won't be possible to pass unknown chart type here since it will throw a TS error.

Reply:
Gotcha, yeah this actually seems better then